### ISSUE TITLE
Updating to v1.5.0 causes ReferenceError in NextJS builds

### ISSUE THREAD

**Issue by WolfyUK** [hCaptcha/react-hcaptcha#176]
The just-released [v1.5.0](https://github.com/hCaptcha/react-hcaptcha/releases/tag/1.5.0) broke our builds with the following error:

```
info  - Compiled successfully
info  - Collecting page data...
ReferenceError: require is not defined in ES module scope, you can use import instead
This file is being treated as an ES module because it has a '.js' file extension and '/home/runner/work/xxx/node_modules/@hcaptcha/react-hcaptcha/package.json' contains "type": "module". To treat it as a CommonJS script, rename it to use the '.cjs' file extension.
    at file:///home/runner/work/xxx/node_modules/@hcaptcha/react-hcaptcha/dist/index.js:3:31
    at ModuleJob.run (node:internal/modules/esm/module_job:193:25)
    at async Promise.all (index 0)
    at async ESMLoader.import (node:internal/modules/esm/loader:530:24)
    at async importModuleDynamicallyWrapper (node:internal/vm/module:4[38](https://github.com/yyy/xxx/actions/runs/000/jobs/111#step:10:39):15)

> Build error occurred
Error: Failed to collect page data for /abc
    at /home/runner/work/xxx/node_modules/next/dist/build/utils.js:960:15 {
  type: 'Error'
}
Error: Process completed with exit code 1.
```

Are there any release notes for the upgrade from 1.4.4 to 1.5.0?

**Comment by altechzilla** [comment]
Similar problem: `Attempted import error: '@hcaptcha/react-hcaptcha' does not contain a default export`

**Comment by yonigoldberg** [comment]
+1

**Comment by numaga94** [comment]
+1

**Comment by zoryana94** [comment]
Hi @WolfyUK @altechzilla @yonigoldberg @numaga94,

This issue should be fixed now in [our latest 1.6.0 release.](https://github.com/hCaptcha/react-hcaptcha/releases/tag/1.6.0)
Please test it from your side and let us know if any further updates are needed.

Sorry for inconvenience. And thank you a lot for helping us to deliver better experiences! 

Best Regards,
IMI Dev Team
